### PR TITLE
RDAPI-19335 migrate 77 Apr2019 Release notes

### PR DESCRIPTION
**Pull request by andreamussap**
## Describe the changes

Migrate 7.7 2019 RN to opendocs - https://techweb.axway.com/jira/browse/RDAPI-19335

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is clear and complete
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links
* [x] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by alexearnshaw** [comment]
@andreamussap I didn't get time to review this today, will do monday. Also, the checklist reminded me ;) - these 2019 RN will also need to be added as new collections to Netlify CMS

**Comment by andreamussap** [comment]
> @andreamussap I didn't get time to review this today, will do monday. Also, the checklist reminded me ;) - these 2019 RN will also need to be added as new collections to Netlify CMS

I'll do it now

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+{
+"title": "API Gateway 7.7 Apr19 Release Notes",
+"linkTitle": "API Gateway 7.7 Apr19",
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: As per recent discussions lets change both to 7.7 April 2019

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+"weight": "20",
+"date": "2020-03-26",
+"description": "Learn about the new features and enhancements in this release of API Gateway."
+}
+
+## Summary
+
+API Gateway is available as a software installation or a virtualized deployment in Docker containers.
+
+The software installation is available on Linux. For more details on supported platforms for software installation, see *API Gateway Installation Guide*.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: Wondering if we should actually just remove the sentences that contained links altogether?

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+This release has the following limitations:
+
+### Elastic topology container deployment
+
+When using an elastic container deployment:
+
+* Traffic monitor data for a specific API Gateway instance does not persist in the event of that instance container stopping. However, you can redirect the trace and traffic logs to `stdout` instead of to separate files, which allows the logs to be read directly from each container by an external logging service.
+* Distributed Ehcache is not supported. However, you can use Apache Cassandra as a distributed data store.
+* To upgrade from versions earlier than 7.6.2 to version 7.8, you must first upgrade to a 7.8 classic deployment and then migrate to an elastic container deployment.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: 7.8 => 7.7

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+For more details, see the *API Gateway Container Deployment Guide*.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: Remove the sentence altogether?

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+
+### Other deployment options
+
+This release is not available as a virtual appliance or as a managed service on Axway Cloud.
+
+## Deprecated features
+
+As part of our software development life cycle we constantly review the core API Management products and related components. As part of this review, API Tester has been deprecated in this release. This standalone tool is no longer supported and will be removed from the next major release (7.8). It is recommended to use alternative tools, such as Postman or SoapUI.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: next major release (7.8) - remove 7.8

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+* CVE-2013-5802
+* CVE-2013-5803
+* CVE-2013-5804
+* CVE-2013-5823
+* CVE-2013-5825
+* CVE-2013-5830
+
+## Removed features
+
+In our efforts to continually upgrade our products in response to the needs of our customers’ IT environments, Axway occasionally discontinues support for some capabilities. API Gateway 7.6.2 is the last release that includes the following capabilities, which have been removed from the 7.8 release:
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: 7.8 => 7.7

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+
+Documentation might contain references to removed features (for example, hardware or virtual appliances, or Windows support). This does not mean that the removed features are still supported, and the references should be ignored.
+
+### Team development – Conflicts when adding dependencies between template projects
+
+In Policy Studio, if you create a new common project from the Common Project template and a new API project from the API Project template and you try to add the API project as a dependent project of the common project, a conflict occurs.
+
+This is due to an issue with the Axway PassPort repositories containing conflicting (randomly generated) password fields in the common and API projects.
+
+As a workaround, use the Entity Explorer tool to set the value of the field `FIELD_KEYSTORE_PASSWORD` to the same value in both the common and API projects. For more information on using the Entity Explorer, see the *API Gateway Developer Guide*.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_ReleaseNotes.md]: Remove the sentence that refs the Dev Guide?

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_fixed_issues.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
+{
+"title": "API Gateway 7.7 Apr19 fixed issues",
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIGatewayApr19_fixed_issues.md]: 7.7 April 2019

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 9]
```diff
@@ -0,0 +1,231 @@
+{
+"title": "API Manager 7.7 Apr19 Release Notes",
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: 7.7 April 2019

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+  "weight": "60",
+  "date": "2020-03-26",
+  "description": "Learn about the new features and enhancements in this release of API Manager."
+}
+
+## Summary
+
+API Manager is a licensed product running on top of API Gateway, and has the same deployment options as API Gateway: software installation, or virtualized deployment in Docker containers. For more information on API Gateway, see the API Gateway Release Notes.
+
+The software installation is available on Linux. For more details on supported platforms for software installation, see *System requirements* in the *API Gateway Installation Guide*.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: Same comment about removing the sentences that had links. Might be better than having italicized refs? Or should we just put in working links (I know we said we would remove them and not fix them, but actually there's only a few involved so we could just fix them?) Wdyt?

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+
+This release has the following limitations:
+
+### Elastic topology container deployment
+
+When using an elastic container deployment:
+
+* Traffic monitor data for a specific API Gateway instance does not persist in the event of that instance container stopping. However, you can redirect the trace and traffic logs to `stdout` instead of to separate files, which allows the logs to be read directly from each container by an external logging service.
+* Distributed Ehcache is not supported. However, you can use Apache Cassandra as a distributed data store.
+* To upgrade from versions earlier than 7.6.2 to version 7.8, you must first upgrade to a 7.8 classic deployment and then migrate to an elastic container deployment.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: 7.8 => 7.7

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+
+For more details, see the *API Gateway Container Deployment Guide*.
+
+### Other deployment options
+
+This release is not available as a virtual appliance or as a managed service on Axway Cloud.
+
+## Deprecated features
+
+As part of our software development life cycle we constantly review the core API Management products and related components. As part of this review, API Tester has been deprecated in this release. This standalone tool is no longer supported and will be removed from the next major release (7.8). It is recommended to use alternative tools, such as Postman or SoapUI.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: 7.8 => remove

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 13]
```diff
@@ -0,0 +1,231 @@
...
+* CVE-2013-5802
+* CVE-2013-5803
+* CVE-2013-5804
+* CVE-2013-5823
+* CVE-2013-5825
+* CVE-2013-5830
+
+## Removed features
+
+In our efforts to continually upgrade our products in response to the needs of our customers’ IT environments, Axway occasionally discontinues support for some capabilities. API Gateway 7.6.2 is the last release that includes the following capabilities, which have been removed from the 7.8 release:
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: 7.8 => 7.7

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 14]
```diff
@@ -0,0 +1,231 @@
...
+The following are known issues for this release of API Manager.
+
+### Documentation might contain references to removed features
+
+Documentation might contain references to removed features (for example, hardware or virtual appliances, or Windows support). This does not mean that the removed features are still supported, and the references should be ignored.
+
+### Cassandra synchronization in multi-datacenter environments
+
+In multi-datacenter environments with Cassandra read/write consistency set to local quorum, there is a small risk of configuration corruption if the event triggering API Manager to load a configuration change happens before the configuration replication to the other datacenter is complete. Changing the polling time as described in
+*Configure API Management in multiple datacenters* in the *API Gateway Installation Guide* reduces this risk, but does not remove it completely.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: fix link? Or rewrite to remove the refs completely?

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md [hunk 15]
```diff
@@ -0,0 +1,231 @@
...
+
+### RAML import does not support references to external files
+
+Importing RAML version 0.8 or RAML version 1.0 files that include references to external files is currently not supported.
+
+Related issues: RDAPI-10356
+
+### Upgrade from API Manager 7.3.0 not supported
+
+Upgrading API Manager version 7.3.0 to version 7.8 is not supported.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_ReleaseNotes.md]: 7.8 => 7.7

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_fixed_issues.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
+{
+"title": "API Manager 7.7 Apr19 fixed issues",
+"linkTitle": "API Manager fixed issues",
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/APIManagerApr19_fixed_issues.md]: 7.7 April 2019

File: content/en/docs/apim_relnotes/201904_apimgr_relnotes/_index.md [hunk 17]
```diff
@@ -0,0 +1,7 @@
+{
+"title": "API Gateway and API Manager 7.7 Apr19 Release Notes",
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apimgr_relnotes/_index.md]: 7.7 April 2019

File: content/en/docs/apim_relnotes/201904_apip_relnotes/_index.md [hunk 18]
```diff
@@ -0,0 +1,157 @@
+{
+"title": "API Portal 7.7 Apr19 Release Notes",
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apip_relnotes/_index.md]: 7.7 April 2019

File: content/en/docs/apim_relnotes/201904_apip_relnotes/_index.md [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+"weight":"30",
+"date": "2019-09-07",
+"description": "Learn about the new features and enhancements in this release of API Portal."
+}
+
+## Summary
+
+API Portal provides an API consumer-facing interface that you can customize to match your corporate brand. API Portal is a layered product linked to API Manager, and requires both API Manager and API Gateway. For more information, see the API Gateway and API Manager documentation.
+
+API Portal is available as a software installation or a virtualized deployment in Docker containers. For more information, see the *API Portal Installation and Upgrade Guide*.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apip_relnotes/_index.md]: remove for more info sentence, or fix the links?

File: content/en/docs/apim_relnotes/201904_apip_relnotes/_index.md [hunk 20]
```diff
@@ -0,0 +1,157 @@
...
+
+Using these standard customization options ensures a smooth upgrade process and will enable you to preserve your customizations when you upgrade to future API Portal releases.
+
+## Limitations of this release
+
+This release has the following limitations:
+
+* This release is not available as a virtual appliance, or as a managed service on Axway Cloud.
+* The ready-made API Portal Docker image is strictly for development environments only, and is not recommended for use in production environments. You must use the Dockerfile to build and run API Portal containers in production environments.
+* Upgrade to API Portal 7.8 is supported from API Portal 7.6.2 only. To upgrade from earlier versions, you must first upgrade to 7.6.2.
```
Comment (alexearnshaw) [inline content/en/docs/apim_relnotes/201904_apip_relnotes/_index.md]: 7.8 => 7.7

File: static/admin/config.js [hunk 21]
```diff
@@ -245,6 +245,12 @@ const collections = [{
   label_singular: 'page in glossary section',
   description: 'Glossary for API Management and AMPLIFY Central.',
   folder: 'content/en/docs/glossary',
+}, {
+  ...docsDefaults('apim_relnotes/201904_apimgr_relnotes', 'docbook/images/release_notes'),
+  name: '201904_apimgr_relnotes',
+  label: 'API Gateway and API Manager 7.7 Apr19 release notes',
```
Comment (alexearnshaw) [inline static/admin/config.js]: 7.7 April 2019

File: static/admin/config.js [hunk 22]
```diff
@@ -245,6 +245,12 @@ const collections = [{
...
+  label_singular: 'page in APIG/APIM 7.7 7.7 Apr19 release note section',
```
Comment (alexearnshaw) [inline static/admin/config.js]: 7.7 is duplicated

File: static/admin/config.js [hunk 23]
```diff
@@ -257,6 +263,13 @@ const collections = [{
   label: 'API Gateway and API Manager 7.7 Mar20 release notes',
   label_singular: 'page in APIG/APIM 7.7 Mar20 release note section',
   description: 'Release notes for API Gateway and API Manager 7.7 Mar20 release.',
+}, {
+  ...docsDefaults('apim_relnotes/201904_apip_relnotes', 'docbook/images/release_notes'),
+  name: '201904_apip_relnotes',
+  label: 'API Portal 7.7 Apr19 release notes',
```
Comment (alexearnshaw) [inline static/admin/config.js]: 7.7 April 2019

**Pull request merged**